README.md

```diff
@@ -19,3 +19,15 @@ The following shortcodes are either ported over from WordPress.com or are create
 The following custom WP-CLI commands exist in this plugin:
 
 * `wpcom-compat import-protected-embeds` - Imports "protected embeds" from a CSV file into the database table `protected_embeds`.
+
+## Convert plain text URLs in `post_content` to links on display.
```

Comment:
Let's create this as a `## Filters` section, and add `wpcom_make_content_clickable()` as a sub-section to that.  That way it's more semantic and other filters can be added there as well.